app/controllers/people_controller.rb

```diff
@@ -5,20 +5,22 @@ class PeopleController < ApplicationController
 
   def signin
     #This is the sign-in sheet, not anything about authentication
-    @people = Person.active
-    @department = Department.where(name: "Police")&.first
+    @department = Department.where(shortname: "BAUX")&.first
```

Comment:
Actually, this will never fail, it will just return nil. So, yes, better to remove the `&`